toonz/sources/common/tfx/ttzpimagefx.cpp

```diff
@@ -5,17 +5,13 @@
 
 #include "ttzpimagefx.h"
 
-#ifdef LINUX
-#define strtok_s strtok_r
-#endif
-
 //**********************************************************************************************
 //    Global  functions
 //**********************************************************************************************
 
 void parseIndexes(string indexes, vector<string> &items)
 {
-#ifndef MACOSX
+#ifdef WIN32
```

Comment:
Currently `WIN32` is being checked for all over.

Changes to existing conventions better be handled outside small patches like this.